crates/superposition/src/organisation/handlers.rs

```diff
@@ -0,0 +1,148 @@
+};
+use actix_web::{
+    get, post,
+    web::{self, Json, Query},
+    HttpResponse, Scope,
+};
+use chrono::Utc;
+use diesel::prelude::*;
+use idgenerator::IdInstance;
+use service_utils::service::types::DbConnection;
+use superposition_types::{
+    custom_query::PaginationParams, result as superposition, PaginatedResponse,
+};
+
+use superposition_types::database::{
+    models::organisation::Organisation, schema::organisation::dsl::organisation,
+};
+
+pub fn endpoints() -> Scope {
+    Scope::new("")
+        .service(create_organisation)
+        .service(list_organisations)
+        .service(get_organisation)
+}
+
+#[post("")]
+pub async fn create_organisation(
+    req: web::Json<CreateOrganisationRequest>,
+    db_conn: DbConnection,
+) -> superposition::Result<HttpResponse> {
+    let DbConnection(mut conn) = db_conn;
+    
+    // Generating a numeric ID from IdInstance and prefixing it with `orgid`
+    let numeric_id = IdInstance::next_id();
+    let org_id = format!("orgid{}", numeric_id);
+    let now = Utc::now().naive_utc();
+    
+    let new_org = Organisation {
+        id: org_id.clone(),
+        country_code: req.country_code.clone(),
+        contact_email: req.contact_email.clone(),
+        contact_phone: req.contact_phone.clone(),
+        created_by: req.created_by.clone(),
+        admin_email: req.admin_email.clone(),
+        status: req.status,
+        sector: req.sector.clone(),
+        created_at: now,
+        updated_at: now,
+        updated_by: req.created_by.clone(),
+    };
+
+    diesel::insert_into(organisation)
+        .values(&new_org)
+        .execute(&mut conn)
+        .map_err(|e| {
+            log::error!("Failed to insert new organisation: {:?}", e);
+            superposition::AppError::UnexpectedError(anyhow::anyhow!(
+                "Failed to create organisation"
+            ))
+        })?;
+
+    let mut http_resp = HttpResponse::Created();
+    Ok(http_resp.json(CreateOrganisationResponse { org_id }))
+}
+
+#[get("/{org_id}")]
+pub async fn get_organisation(
+    org_id: web::Path<String>,
+    db_conn: DbConnection,
+) -> superposition::Result<HttpResponse> {
+    let DbConnection(mut conn) = db_conn;
+    
+    let org = organisation
+        .find(org_id.as_str())
+        .first::<Organisation>(&mut conn)
+        .map_err(|e| {
+            log::error!("Failed to fetch organisation {}: {:?}", org_id, e);
+            match e {
+                diesel::result::Error::NotFound => {
+                    superposition::AppError::NotFound(format!(
+                        "Organisation {} not found",
+                        org_id
+                    ))
+                }
+                _ => superposition::AppError::UnexpectedError(anyhow::anyhow!(
+                    "Failed to fetch organisation"
+                )),
+            }
+        })?;
+
+    Ok(HttpResponse::Ok().json(OrganisationResponse::from(org)))
```

Comment:
if we can remove the `OrganisationResponse` and directly use `Organisation` type then we can simplify it here to:
```suggestion
    Ok(HttpResponse::Ok().json(org))
```